README.rst

```diff
@@ -66,29 +66,37 @@ Behind the scene, it uses `Ziggurat-Foundations`_ and `Authomatic`_.
 .. end-badges
 
 
-REST API Documentation
-======================
+Documentation
+=============
+
+The REST API documentation is auto-generated and served under ``{HOSTNAME}/api/`` using Swagger-UI with tag ``latest``.
```

Comment:
Did you mean `https://{HOSTNAME}/api`?

Reply:
Protocol is not necessarily https. 
I mostly want to convey the idea that it is available under path `/api` of wherever the app instance what configured to run. 
Is it clearer with `{MAGPIE_URL}/api` ?

Reply:
Yeah, mentioning URL will include whatever the protocol is :D